crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -260,29 +237,8 @@ impl VM {
             .try_into()
             .map_err(|_| VMError::VeryLargeNumber)?;
 
-        let minimum_word_size = (size
-            .checked_add(WORD_SIZE)
-            .ok_or(VMError::Internal(
-                InternalError::ArithmeticOperationOverflow,
-            ))?
-            .saturating_sub(1))
-            / WORD_SIZE;
-
-        let memory_expansion_cost =
-            current_call_frame
-                .memory
-                .expansion_cost(dest_offset.checked_add(size).ok_or(VMError::Internal(
-                    InternalError::ArithmeticOperationOverflow,
-                ))?)?;
-
-        let minimum_word_size_cost = gas_cost::CODECOPY_DYNAMIC_BASE
-            .checked_add(minimum_word_size.into())
-            .ok_or(VMError::GasCostOverflow)?;
-        let gas_cost = gas_cost::CODECOPY_STATIC
-            .checked_add(minimum_word_size_cost)
-            .ok_or(VMError::GasCostOverflow)?
-            .checked_add(memory_expansion_cost)
-            .ok_or(VMError::GasCostOverflow)?;
+        let gas_cost = codecopy_gas_cost(current_call_frame, size, dest_offset)
```

Comment:
Use the namespace.
```suggestion
        let gas_cost = gas_cost::codecopy_gas_cost(current_call_frame, size, dest_offset)
```

Reply:
Al these are done in [Use explicit gas_cost:: instead of importing funcs](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1138/commits/307eb7b02dfdc48d9b1e66b8350ce50b2f7b89e0)